unit-tests/func/func-common.h

```diff
@@ -94,3 +135,30 @@ inline stream_profile find_confidence_corresponding_to_depth( rs2::depth_sensor
     REQUIRE( confidence_profile != stream_profiles.end() );
     return *confidence_profile;
 }
+
+
+bool get_profile_by_stream_parameters( rs2::depth_sensor ds,
```

Comment:
Why only `depth_sensor`?

Reply:
What about `fps`? `stream_index` (for D400)?

Reply:
> Why only `depth_sensor`?, changed to sensor

Reply:
> What about `fps`? `stream_index` (for D400)?

Added with default values